ClLike/cl_like/power_spectrum.py

```diff
@@ -187,6 +219,8 @@ def _get_pk_data(self, cosmo):
                     if op1 != op2:
                         comb_21 = op2+op1
                         pkd[f'pk_{comb_21}'] = pkd[f'pk_{comb_12}']
+            if self.bias_model == 'BaccoPT':
+                pkd['pk_mm_sh_sh'] = ptc.get_pk('mm_sh_sh', pnl=pkmm, cosmo=cosmo)
```

Comment:
@damonge, @matteozennaro would this work if, instead of adding `mm_sh_sh` we use `ww`? That way it would fit nicely with BLCDM. It can be left for the future.

Reply:
I think it could work, only thing is we don't exactly store there the ww Pk... could this be confusing? Anyways, no strong preferences from my part

Reply:
Let's leave it like this for now